components/MonitoringUserSetupInstructions.tsx

```diff
@@ -74,7 +74,7 @@ DECLARE
   prepared_params text;
   result text;
 BEGIN
-  SELECT regexp_replace(query, ';+\s*\Z', '') INTO prepared_query;
+  SELECT regexp_replace(query, ';+\\s*\\Z', '') INTO prepared_query;
```

Comment:
Huh, interesting. I wonder if this has caused problems in the past where people have a bad function definition.

Reply:
Yeah. Unfortunately, I'm not sure if there's a good way to detect that, but I have not seen any support requests around this (and it should break log-based EXPLAIN if we don't strip the trailing semicolon correctly.). We could announce it in e.g., the newsletter, or the next collector release notes, but given the limited impact, I think we can just make the change silently.